spec/models/preserved_copy_spec.rb

```diff
@@ -156,35 +103,24 @@
   end
 
   describe '#update_status' do
-    let(:pc) do
-      # using create here, because if the object has never been saved, #changed? will always return true
-      PreservedCopy.create(
-        preserved_object_id: preserved_object.id,
-        endpoint_id: endpoint.id,
-        version: 0,
-        status: PreservedCopy::INVALID_MOAB_STATUS,
-        size: 1
-      )
-    end
-
     it 'does nothing if the status has not changed' do
       ran_the_block = false
-      pc.update_status(PreservedCopy::INVALID_MOAB_STATUS) { ran_the_block = true }
-      expect(pc.status).to eq PreservedCopy::INVALID_MOAB_STATUS
+      preserved_copy.update_status(preserved_copy.status) { ran_the_block = true }
+      expect(preserved_copy.status).to eq 'validity_unknown'
       expect(ran_the_block).to eq false
-      expect(pc.changed?).to eq false
+      expect(preserved_copy.status_changed?).to eq false
```

Comment:
By definition, AR's `changed?` will be true every time any `<attribute>_changed?` is true.  So I am testing the more specific case that we care about (the status).  I see your point though am OK changing it back or testing both, if your prefer.

Otherwise yeah, I would support the class revision work as a follow-on.

Reply:
yeah, if you wouldn't mind testing both, that'd be great, filing a ticket for the other part now...

Reply:
https://github.com/sul-dlss/preservation_catalog/issues/801 for dropping `PreservedCopy#update_status` and putting what it does inline in `MoabValidationHandler`.

Reply:
AFAICT we're not actually saving if changed anywhere (we're only using changed in two places, both in the tests, i'm pretty sure, see #801).  but we may actually have done that at one point.  anyway, happy to not be doing that now if we had been.